content/resources/operator/_index.md

```diff
@@ -81,7 +81,106 @@ Blood Pressure Reading:
 
 One of the goals when providing an external reference is to figure out the level of specificity when breaking down a property name that contains multiple concepts.  The question is whether the new references should be created with very specific designations (This is known as pre-coordination).   This option would likely create the need for the request of new terms in the external standard if the term is not in existence. The other question is, should the use of multiple terms that already exist in an external standard be used (This is known as post-coordination)?  The best practice adopted by CTDS is to use specificity whenever corresponding terms are available in the external standard.  However, If specific terms are not available, use generality by creating multiple terms that already exist in an external standard.  For instance, if grapefruit juice is a property of interest and it is not found in the external reference, but grapefruit and juice are found individually, then using the individual properties is the preferred method.
 
-## 4. Programs and Projects
+
+## 4. Authentication Methods
+
+The following methods of authentication are supported in Gen3. They are listed in order of preference from OCC perspective, and technological and governance considerations are outlined below.
+
+### eRA Commons
+
+Authentication system developed by NIH for managing for research grants.
```

Comment:
I think it should be “for managing of research grants” or “for the management of research grants”

Reply:
Thanks! Done in 02e5565.